README.md

```diff
@@ -3,80 +3,90 @@
 
 # Chatmail servers for secure instant messaging 
 
-Chatmail servers are minimal interoperable e-mail routing machines designed for:
+Chatmail servers are interoperable email routing machines optimized for:
 
-- **Convenience:** Instant onboarding, with optional Google/Apple/Huawei push notifications
+- **Convenience:** Low friction instant onboarding
 
-- **Privacy:** Just login, no questions asked, no name, numbers or e-mail needed
+- **Privacy:** No name, telephone numbers, or email required to sign-up permitting you to create unique and disposable identities. Push notification architecture is privacy-preserving. (Apple, Google, and Huawei)
```

Comment:
when making a bullet-point list i suggest to have each point consume similarly many characters. 

```suggestion
- **Privacy:** No name, phone numbers,  email required or collected 
- **Instant:** Privacy-preserving push notifications for Apple, Google, and Huawei
```